GUIDE_SORMAS2SORMAS_CERTIFICATE.md

```diff
@@ -14,31 +15,36 @@ See [Installing Java](SERVER_SETUP.md#java-11)
 ### Using the certificate generation script
 
 1. Run ``bash ./generate-cert.sh``
-2. If the ``sormas2sormas`` directory is not found, you will be prompted to provide its path.
-3. If the ``SORMAS_PROPERTIES`` environment variable is not available, the script will search for the ``sormas.properties`` 
-file in ``/opt/domains/sormas/sormas.properties`` by default. If it is not found there, you will be prompted to provide 
-the path to the ``sormas.properties`` file.
-4. For the generation of the certificate, the following data is needed: a password, a *Common Name* (CN) 
-    and an *Organization* (O). These may be set in environment variables (recommended), or provided 
-    manually as the script executes.
-    * The password environment variable should be named ``SORMAS_S2S_CERT_PASS``. Please note that the password has to be 
-    at least 6 characters, or you will be prompted for a new one.
-    * the *Common Name* environment variable should be named ``SORMAS_S2S_CERT_CN``.<br/>
+2. If the ``SORMAS2SORMAS_DIR`` environment variable is not available, the script will search for ``/opt/sormas2sormas`` by default. 
+If it is not found there, you will be prompted to provide the pat to the *sormas2sormas* directory.
+3. If the ``SORMAS_DOMAIN_DIR`` environment variable is not available, the script will search for ``/opt/domains/sormas`` by default. 
+If it is not found there, you will be prompted to provide the path to the *sormas domain directory*.
+4. For the generation of the certificate, the following data is needed:
+   an identifier of the *Organization* (e.g. a UUID), the name of the *Organization*, the host name of the SORMAS server, the **https** port of the server,
+   a password for the certificate key store and a password for the REST user to be used when sharing data through the REST api.
+   These may be set in environment variables (recommended), or provided manually as the script executes.
+
+    * the identifier of the *Organization* environment variable should be named ``SORMAS_ORG_ID``. 
+    This variable is also used as *Common Name* (CN) of the certificate<br/>
     **Important**: for Germany, this value should be the SurvNet Code Site. <br/>
```

Comment:
Probably better: "for Germany, this value should be the SORMAS SurvNet Code Site"
(The regular SurvNet Code Site would be 1.03.1.01)